novawallet/Modules/Notifications/NotificationHandlers/MultisigPayload/MultisigPayload+Rejected.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+extension MultisigPayloadCodingKey {
```

Comment:
I was hard to understand how this work as generic parameter for the model. I think it is more readable to create separate models for each payload type. WDYT?